.dotnet/src/Custom/OpenAIClientOptions.cs

```diff
@@ -18,4 +18,6 @@ public partial class OpenAIClientOptions : ClientPipelineOptions
     /// An optional application ID to use as part of the request User-Agent header.
     /// </summary>
     public string ApplicationId { get; init; }
+
+    public ServiceLatencyTierChoice? LatencyTierChoice { get; init; }
```

Comment:
Something that makes me a little nervous is that this currently only works for chat completions and putting it in the OpenAIClientOptions could give the impression that it works everywhere. I can imagine OpenAI extending this to other endpoints in the future, but I'm also worried that the resulting API might not be exactly identical, and then we wouldn't be able to reuse this property.

I agree that putting this property in ChatCompletionOptions feels a little odd for an API with this kind of functionality, but it also feels to me like other alternatives don't provide a significant improvement either. This is making lean towards playing it safe and keeping it in the ChatCompletionOptions to at least keep it consistent with the REST API.